.github/workflows/rust.yml

```diff
@@ -14,62 +14,57 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
```

Comment:
❌ Needs a version tag after the hash.

Reply:
Fixed in [4588d8f](https://github.com/bitwarden/bitwarden-russh/pull/5/commits/4588d8f4afd966fbd7be259cf3c24dd4999c3a97)